### ISSUE TITLE
geoLocation issues

### ISSUE THREAD

**Issue by bojanjukic** [nitaliano/react-native-mapbox-gl#1107]
Hi!

Unfortunately I have a issue with geoLocation on my iOS devices. Sometimes it centers correctly and sometimes the user location is positioned on the right bottom corner. I haven't found a pattern. 

When component mounts (Map.js)
`async componentWillMount () {
        navigator.geolocation.getCurrentPosition(
            (position) => {
                this.setState({
                    latitude: position.coords.latitude,
                    longitude: position.coords.longitude,
                    error: null,
                });
            },
            (error) => this.setState({ error: error.message }),
            { enableHighAccuracy: true, timeout: 1000, maximumAge: 1000 },
        );`

```
componentDidMount () {
            this._locationWatchID = navigator.geolocation.watchPosition(
                (position) => {
                    this.onLocationChange(position);
                },
                (error) => this.setState({ error: error.message }),
                { enableHighAccuracy: true, timeout: 1000, maximumAge: 1000, distanceFilter: 10 },
            );
}

```

```
onLocationChange (position) {
        const coord = [position.coords.longitude, position.coords.latitude];
        this.props.onLocationChange(coord);

        this.setState({
            currentPosition: MapboxGL.geoUtils.makePoint(coord),
        });
    }
```

If you need any information for better troubleshooting, I will be happy to provide it. Thank you. 

![image uploaded from ios 1](https://user-images.githubusercontent.com/33836904/37290839-9813b4f0-260d-11e8-9929-b031c800c67d.jpg)
![image uploaded from ios](https://user-images.githubusercontent.com/33836904/37290840-98438356-260d-11e8-8fc2-4a913357533b.jpg)

**Comment by nitaliano** [comment]
What version of the SDK are you using? It could also be the geolocation from react native it's not as accurate as the native location managers

**Comment by bojanjukic** [comment]
The strange thing: If the issue happens, then the location is always in the bottom right corner. So accurancy should not be the problem. 

"@mapbox/react-native-mapbox-gl": "^6.0.3",
"mapbox": "^1.0.0-beta9",

**Comment by nitaliano** [comment]
Try updating to 6.1.0 we fixed an overflow bug where the map would randomly overflow it's parent, you could be running into that

**Comment by henrikra** [comment]
Related https://github.com/mapbox/react-native-mapbox-gl/issues/777

**Comment by nitaliano** [comment]
I'm closing this out since I believe it's related to the overflow bug that is fixed in `6.1.0`

**Comment by jaydesai76002** [comment]
> I'm closing this out since I believe it's related to the overflow bug that is fixed in `6.1.0`

hy can you help me on MapboxGL.geoUtils.makePoint()
